src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2121,263 @@ definitions:
         description: path to patch
       value:
         type: string
-        description: value of path
\ No newline at end of file
+        description: value of path
+  
+  subscribe_user_to_alert_body:
+    type: object
+    description: The definition for subscribing emailId's to query or schedule IDs for a given alertType
+    required:
+      - assetId
+      - alertType
+      - subscriptions
+    properties:
+      assetId:
+        type: string
+        description: The query or schedule ID that the alert is created for and subscribed to.
```

Comment:
These properties are for the _request_ payload, right? These aren't the result of a GET request, so I'm not sure why we're using past tense to describe this.

For all _request_ payload properties, we need to describe them in terms of what the user needs to provide as a value, and/or how they affect the result of the operation. E.g.: "The ID for the query or schedule that you want the alert to track."

For all _response_ payload properties, we need to describe them in terms of what the values mean as part of an existing/created entity. E.g.: "The ID for the query or schedule that the alert is tracking."